docs/spark/tutorials/hdinsight-deployment.md

```diff
@@ -0,0 +1,135 @@
+
+   * `microsoft-spark-<spark_majorversion.spark_minorversion.x>-<spark_dotnet_version>.jar`: This jar is included as part of the [Microsoft.Spark](https://www.nuget.org/packages/Microsoft.Spark/) NuGet package and is colocated in your app's build output directory.
+   * `<your app>.zip`
+   * Files (like dependency files or common data accessible to every worker) or Assemblies (like DLLs that contain your user-defined functions or libraries that your `app` depends on) to be placed in the working directory of each executor.
+
+## Deploy to Azure HDInsight Spark
+
+[Azure HDInsight Spark](https://docs.microsoft.com/azure/hdinsight/spark/apache-spark-overview) is the Microsoft implementation of Apache Spark in the cloud that allows users to launch and configure Spark clusters in Azure. You can use HDInsight Spark clusters to process your data stored in [Azure Storage](https://azure.microsoft.com/services/storage/) or [Azure Data Lake Storage](https://docs.microsoft.com/azure/storage/blobs/data-lake-storage-introduction).
+
+> [!NOTE]
+> Azure HDInsight Spark is Linux-based. If you are interested in deploying your app to Azure HDInsight Spark, make sure your app is .NET Standard compatible and that you use the [.NET Core compiler](https://dotnet.microsoft.com/download) to compile your app.
+
+### Deploy Microsoft.Spark.Worker
+
+This step is only required once for your cluster.
+
+Run `install-worker.sh` on the cluster using [HDInsight Script Actions](https://docs.microsoft.com/azure/hdinsight/hdinsight-hadoop-customize-cluster-linux).
+
+|Setting|Value|
+|-------|-----|
+|Script type|Custom|
+|Name|Install Microsoft.Spark.Worker|
+|Bash script URI|The URI to which you uploaded `install-worker.sh`. For example, `adl://<cluster name>.azuredatalakestore.net/<some dir>/install-worker.sh`|
+|Node type(s)|Worker|
+|Parameters|Parameters to `install-worker.sh`. For example, if you uploaded `install-worker.sh` to Azure Data Lake then it would be `azure adl://<cluster name>.azuredatalakestore.net/<some dir>/Microsoft.Spark.Worker.<release>.tar.gz /usr/local/bin`.|
+
+![Script Action Image](./media/hdinsight-deployment/deployment-hdi-action-script.png)
+
+## Run your app
+
+You can submit your job to Azure HDInsight using `spark-submit` or Apache Livy.
+
+### Use spark-submit
+
+You can use the [spark-submit](https://spark.apache.org/docs/latest/submitting-applications.html) command to submit .NET for Apache Spark jobs to Azure HDInsight.
+ 
+1. `ssh` into one of the head nodes in your cluster.
+
+1. Run `spark-submit`:
+
+   ```bash
+   spark-submit \
+   --master yarn \
+   --class org.apache.spark.deploy.DotnetRunner \
+   --files <comma-separated list of assemblies that contain UDF definitions, if any> \
+   adl://<cluster name>.azuredatalakestore.net/<some dir>/microsoft-spark-<spark_majorversion.spark_minorversion.x>-<spark_dotnet_version>.jar \
```

Comment:
These adl:// file paths mean we are using ADLS Gen 1? We should default to blob store or ADLS Gen 2 to be "current"